frontend/packages/ceph-storage-plugin/src/queries/ceph-queries.ts

```diff
@@ -50,12 +53,32 @@ export const DATA_RESILIENCY_QUERY = {
 export const UTILIZATION_QUERY = {
   [StorageDashboardQuery.CEPH_CAPACITY_USED]:
     'sum(kubelet_volume_stats_used_bytes * on (namespace,persistentvolumeclaim) group_left(storageclass, provisioner) (kube_persistentvolumeclaim_info * on (storageclass)  group_left(provisioner) kube_storageclass_info {provisioner=~"(.*rbd.csi.ceph.com)|(.*cephfs.csi.ceph.com)"}))',
-  [StorageDashboardQuery.UTILIZATION_IOPS_QUERY]:
-    '(sum(rate(ceph_pool_wr[1m])) + sum(rate(ceph_pool_rd[1m])))',
-  [StorageDashboardQuery.UTILIZATION_LATENCY_QUERY]:
-    '(quantile(.95,(cluster:ceph_disk_latency:join_ceph_node_disk_irate1m)))',
-  [StorageDashboardQuery.UTILIZATION_THROUGHPUT_QUERY]:
-    '(sum(rate(ceph_pool_wr_bytes[1m]) + rate(ceph_pool_rd_bytes[1m])))',
+  [StorageDashboardQuery.UTILIZATION_IOPS_READ_QUERY]: {
+    query: 'ceil(sum(rate(ceph_pool_rd[1m])))',
+    desc: 'Reads',
+  },
+  [StorageDashboardQuery.UTILIZATION_IOPS_WRITE_QUERY]: {
+    query: 'ceil(sum(rate(ceph_pool_wr[1m])))',
+    desc: 'Writes',
+  },
+  [StorageDashboardQuery.UTILIZATION_LATENCY_READ_QUERY]: {
+    query:
+      'sum by (namespace) (topk by (ceph_daemon) (1, label_replace(label_replace(ceph_disk_occupation{job="rook-ceph-mgr"}, "instance", "$1", "exported_instance", "(.*)"), "device", "$1", "device", "/dev/(.*)")) * on(instance, device) group_left topk by (instance,device) (1, (rate(node_disk_read_time_seconds_total[1m]) / (clamp_min(rate(node_disk_reads_completed_total[1m]), 1)))))',
```

Comment:
```suggestion
      'sum by (namespace) (
        topk by (ceph_daemon) (
          1, label_replace(
            label_replace(
              ceph_disk_occupation{job="rook-ceph-mgr"}, "instance", "$1", "exported_instance", "(.*)"
            ), "device", "$1", "device", "/dev/(.*)"
          )
        ) * on(instance, device) group_left topk by (instance,device) (
          1, (
            rate(node_disk_read_time_seconds_total[1m])
            /
            (clamp_min(rate(node_disk_reads_completed_total[1m]), 1))
          )
        )
      )',
```
If possible, let's convert this to a multiline for readability.